cross/lame/patches/001-better_check_sse.patch

```diff
@@ -0,0 +1,379 @@
+# Will be fix in lame 3.100
```

Comment:
This patch seemss way larger than it needs to be. 
There are changes on comments, doc changes that we don't care about and a huge chunk is related to changes to GTK+ that doesn't seem to make sense in this context.
Can this be reviewed and minimized to the bare essentials?